otlp/traces.go

```diff
@@ -248,30 +248,24 @@ func shouldTrimTraceId(traceID []byte) bool {
 //
 // See: https://github.com/open-telemetry/opentelemetry-proto/blob/59c488bfb8fb6d0458ad6425758b70259ff4a2bd/opentelemetry/proto/trace/v1/trace.proto#L230
 func evaluateSpanStatus(status *trace.Status) (int, bool) {
+	const okStatus = int(trace.Status_STATUS_CODE_OK)
 	const unsetStatus = int(trace.Status_STATUS_CODE_UNSET)
 	const errorStatus = int(trace.Status_STATUS_CODE_ERROR)
 
 	if status == nil {
 		return unsetStatus, false
 	}
 
-	isError := false
```

Comment:
This entire function reduces to:

```go
func evaluateSpanStatus(status *trace.Status) (int, bool) {
	if status == nil {
		return int(trace.Status_STATUS_CODE_UNSET), false
	}

	return int(status.Code), false
}
```

And since it never returns true, should probably also have the extra bool parameter removed.

Reply:
We still want to return the bool if the status code is ERROR, but I agree this func is a little more complicated than necessary where it makes local constants of the ok, error and unset statuses. I'll refactor to make simpler.